assets/js/helpers/boot.js

```diff
@@ -76,3 +78,16 @@ $(document).ready(function() {
   });
   window.bootTopic();
 });
+
+getScript = function(src, callback) {
```

Comment:
Also calling getScript on our custom script will not work in production because everyting is minified and included into one file called something like `production.12345.js`